src/TextLogging/CMakeLists.txt

```diff
@@ -0,0 +1,9 @@
+# Copyright (C) 2020 Istituto Italiano di Tecnologia (IIT). All rights reserved.
+# This software may be modified and distributed under the terms of the
+# GNU Lesser General Public License v2.1 or any later version.
+
+add_bipedal_locomotion_library(
+  NAME                   TextLogging
+  PUBLIC_HEADERS         include/BipedalLocomotion/TextLogging/Logger.h
+  SOURCES                src/Logger.cpp
+  PUBLIC_LINK_LIBRARIES  spdlog::spdlog)
```

Comment:
Nitpick: Why the names `Textlogging` and a `Logger`? I believe if we are going to use this only for debug prints then `Logger` is too generic for a name.

Reply:
I'm terrible with the names :smiley: . Do you have any suggestions?

Reply:
Now that, I've seen these names, I am not able to think of others ;D

Reply:
I think it's ok.